torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -327,8 +327,11 @@ void IRPrinter::handle(const Allocate* const a) {
 void IRPrinter::handle(const Split* const s) {
   os << "Split: ";
   handle(s->in());
-  os << " axis " << s->axis() << " by factor " << s->factor() << " -> ";
+  os << " axis " << s->axis() << " by factor " << s->factor() << " ->\n";
```

Comment:
Nit, `"\n  ->"` instead of `" ->\n"`
Should follow for merge and reorder too.

Reply:
Sounds good. Btw, I found that this small split makes the IR dumps much more easy to read.